frontend/installation-ui/web/micro-ui-internals/packages/modules/qc/src/pages/employee/FacilityDetails.js

```
@@ -115,37 +117,53 @@ const FacilityDetails = ({t}) => {
 
-      {fetchedData && fetchedData.map((asset) => {
+      {assets && assets.map((asset) => {
         return <Summary
           sectionName={asset?.assetName}
           count={asset?.count}
           specifications={asset?.specifications}
           details={asset?.details}
           items={asset?.items}
+          images={aggregatedAssets.images?.[asset.assetType]}
+          videos={aggregatedAssets.videos?.[asset.assetType]}
         />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)